.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,94 @@
+name: Unit Tests
+
+on: pull_request
+
+jobs:
+  tests:
+    name: "WP: ${{ matrix.wp_version }} - PHP: ${{ matrix.php }} - ES: ${{ matrix.es_version }} (MU: ${{ matrix.multisite }})"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false # do not fail fast, let all the failing tests fail.
+      matrix:
+        php: [8.0]
+        es_version: [7.17.5]
+        multisite: [0]
+        wp_version: ["latest"]
+        include:
+          - php: '7.4'
```

Comment:
Something we're going to have to consider for this plugin is that we're testing different _adapters_ (VIP Enterprise Search, SerachPress) and not necessarily different versions of Elasticsearch. For example, if VIP Enterprise Search only runs on Elasticsearch 7, then we should only test it on that version, otherwise failures might be upstream of our code. Nothing to do for now, but we will probably want to either make separate unit test actions _per adapter_ or we will want to specify which adapter(s) we are testing on each matrix item in future PRs.

Reply:
Agree! The `boostrap.php` file will also need to handle both adapters.